File: AD-HOC/2473.py
sorteados = list(map(int, input().split()));
apostados = list(map(int, input().split()));
qtde = 0;

for i in range(len(sorteados)):
  for j in range(len(apostados)):
    if (apostados[j] == sorteados[i]):
      qtde += 1;

if (qtde == 3):
  print("terno");
elif (qtde == 4):
  print("quadra");
elif (qtde == 5):
  print("quina");
elif (qtde == 6):
  print("sena");
else:
  print("azar");

File: AD-HOC/2473.js
let sorteados = lines.shift().split(' ');
let apostados = lines.shift().split(' ');
let qtde = 0;

for (let i = 0; i < sorteados.length; i++) {
  for (let j = 0; j < apostados.length; j++) {
    if (apostados[j] == sorteados[i]) {
      qtde++;
    }
  }
}

if (qtde == 3) {
  console.log("terno");
} else if (qtde == 4) {
  console.log("quadra");
} else if (qtde == 5) {
  console.log("quina");
} else if (qtde == 6) {
  console.log("sena");
} else {
  console.log("azar");
}
